### PR TITLE
RDoc-1695 Clarified Java and C# Query-syntax tabs as LINQ-syntax and …

### PR DESCRIPTION
**Pull request by serezhub**
…clarified goal of each code sample

### CODE REVIEW

File: Documentation/4.0/Raven.Documentation.Pages/indexes/map-indexes.dotnet.markdown [hunk 1]
```diff
@@ -97,11 +97,12 @@ The part you should pay attention to is `docs.Employees`. This syntax indicates
 Since each index contains a LINQ function, you can combine multiple fields into one.
 
 ### Example I
-
+Index definition with LINQ-syntax:
```
Comment (Danielle9897) [inline Documentation/4.0/Raven.Documentation.Pages/indexes/map-indexes.dotnet.markdown:100]: => Index definitions:

File: Documentation/4.0/Raven.Documentation.Pages/indexes/map-indexes.dotnet.markdown [hunk 2]
```diff
@@ -97,11 +97,12 @@ The part you should pay attention to is `docs.Employees`. This syntax indicates
...
 ### Example I
-
+Index definition with LINQ-syntax:  
 {CODE indexes_7@Indexes/Map.cs /}
-
+  
+Query the index:  
 {CODE-TABS}
-{CODE-TAB:csharp:Query indexes_8@Indexes/Map.cs /}
+{CODE-TAB:csharp:LINQ-syntax indexes_8@Indexes/Map.cs /}
```
Comment (Danielle9897) [inline Documentation/4.0/Raven.Documentation.Pages/indexes/map-indexes.dotnet.markdown:105]: This is the query part, so return the 'Query'

LINQ-syntax  => Query

**Review by Danielle9897 — APPROVED** [review]

File: Documentation/5.2/Raven.Documentation.Pages/indexes/map-indexes.java.markdown [hunk 3]
```diff
@@ -115,11 +115,13 @@ Since each index contains a LINQ function, you can combine multiple fields into
 
 ### Example I
 
+Define the index:  
 {CODE-TABS}
-{CODE-TAB:java indexes_7@Indexes/Map.java /}
+{CODE-TAB:java:LINQ indexes_7@Indexes/Map.java /}
```
[the remark this answers is not in the document]
Comment (serezhub) [reply Documentation/5.2/Raven.Documentation.Pages/indexes/map-indexes.java.markdown]: Done.

File: Documentation/5.2/Raven.Documentation.Pages/indexes/map-indexes.java.markdown [hunk 4]
```diff
@@ -138,11 +140,13 @@ You can read more about analyzers and `Full Text Search` [here](../indexes/using
 
 {INFO/}
 
+Defining the index:
```
[the remark this answers is not in the document]
Comment (serezhub) [reply Documentation/5.2/Raven.Documentation.Pages/indexes/map-indexes.java.markdown]: Done.

File: Documentation/4.0/Raven.Documentation.Pages/indexes/map-indexes.dotnet.markdown (hunk 1 above)
Comment (serezhub) [reply Documentation/4.0/Raven.Documentation.Pages/indexes/map-indexes.dotnet.markdown]: Done.

File: Documentation/4.0/Raven.Documentation.Pages/indexes/map-indexes.dotnet.markdown (hunk 2 above)
Comment (serezhub) [reply Documentation/4.0/Raven.Documentation.Pages/indexes/map-indexes.dotnet.markdown]: Done.

**Pull request merged**